### PR TITLE
Migrate UI to webcomponents

### PR DESCRIPTION
**Pull request by cbadura**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -45,5 +45,17 @@ const plugins = config.plugins.filter((plugin) => !(plugin instanceof ModifyEntr
 
 module.exports = {
   ...config,
-  plugins
+  plugins,
+  output: {
+    uniqueName: 'my-ui',
```
Comment (markuczy) [inline webpack.config.js:50]: Should be `onecx-parameter-ui`

**Review by markuczy — CHANGES_REQUESTED** [review]

File: src/app/onecx-parameter-remote.module.ts [hunk 2]
```diff
@@ -1,27 +1,45 @@
...
+import { initializeRouter, startsWith } from '@onecx/angular-webcomponents'
+import { AngularAuthModule } from '@onecx/angular-auth'
+import { AppEntrypointComponent } from './app-entrypoint.component'
+import { environment } from 'src/environments/environment'
+import { Configuration } from './shared/generated'
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations'
+
+function apiConfigProvider(configService: ConfigurationService, appStateService: AppStateService) {
+  return new PortalApiConfiguration(Configuration, environment.apiPrefix, configService, appStateService)
+}
```
Comment (markuczy) [inline src/app/onecx-parameter-remote.module.ts:26]: Make sure its not duplicated in SharedModule

Comment (cbadura) [reply src/app/onecx-parameter-remote.module.ts:26]: done

File: webpack.config.js (hunk 1 above)
Comment (cbadura) [reply webpack.config.js]: done

**Review by markuczy — APPROVED** [review]

**Review by SchettlerKoehler — APPROVED** [review]

**Pull request merged**